bulkcopy_test.go

```diff
@@ -300,6 +300,14 @@ func TestBulkcopy(t *testing.T) {
 	}
 }
 
+func TestBulkcopyWithGuidConversion(t *testing.T) {
+	testBulkcopy(t, true /*guidConversion*/)
+}
+
+func TestBulkcopyWithoutGuidConversion(t *testing.T) {
```

Comment:
let's keep the test names for the non-guid conversion cases the same as they were before.